onnxruntime/python/tools/transformers/models/llama/llama_parity.py

```diff
@@ -113,13 +122,14 @@ def verify_parity(args: argparse.Namespace, config: LlamaConfig, pt_model: Llama
     if args.execution_provider != "cpu":
         io_binding = add_io_bindings(args, ort_model, inputs)
 
-        io_binding.synchronize_inputs()
+        torch.cuda.synchronize()
```

Comment:
Same comment as before to revert these changes since other EPs need them

Reply:
will do, thanks